apps/src/lib/node/ledger/tendermint_node.rs

```diff
@@ -249,7 +297,7 @@ fn update_tendermint_config(
 
     // In "dev", only produce blocks when there are txs or when the AppHash
     // changes
-    config.consensus.create_empty_blocks = !cfg!(feature = "dev");
+    config.consensus.create_empty_blocks = true; // !cfg!(feature = "dev");
```

Comment:
maybe we should remove the explicit update, so we can change this in tendermint config, or even better add this to our ledger config so we can change it without re-compiling